CRM/UF/Page/Group.php

```diff
@@ -286,8 +284,6 @@ public function edit($id, $action) {
   /**
    * Browse all uf data groups.
    *
-   * @param
-   *
```

Comment:
This one does have a param, but it looks like it's always overridden by the local param with the same name.

Reply:
Good catch, it's called without a param on line 170, so I'll remove the `$action = NULL`.